README.md

```diff
@@ -169,20 +169,29 @@ enable('<YOUR INSTRUMENTATION KEY GOES HERE>')
 raise Exception('Boom!')
 ```
 
-**Logging requests**
+**Integrating with Flask**
 ```python
 from flask import Flask
-from applicationinsights.requests import WSGIApplication
+from applicationinsights.flask.ext import AppInsights
 
-# instantiate the Flask application and wrap its WSGI application
+# instantiate the Flask application
 app = Flask(__name__)
-app.wsgi_app = WSGIApplication('<YOUR INSTRUMENTATION KEY GOES HERE>', app.wsgi_app)
+app.config['APPLICATION_INSIGHTS_KEY'] = '<YOUR INSTRUMENTATION KEY GOES HERE>'
```

Comment:
Flask doesn't translate environment variables into the config automatically. So this comment is not very critical. On some environments in Azure like Web Apps instrumentation key will be set in environment variable `APPINSIGHTS_INSTRUMENTATIONKEY`. Name you use for `config` is different. Would it be more confusing to align with the name of environment variable or beneficial?

Reply:
Might have slightly misunderstood the original intent here so I also added the option to auto-load the value from the OS environment in 157a5b7.